funflow-tutorial/notebooks/Tutorial1.ipynb

```diff
@@ -6,22 +6,22 @@
    "source": [
     "# First steps with `funflow`\n",
     "\n",
+    "\n",
     "## Introduction\n",
     "\n",
-    "`funflow` is a Haskell library for defining and running _workflows_.\n",
+    "`funflow` is a Haskell library for defining and running _workflows_, which here we use synonymously with _data pipelines_.\n",
```

Comment:
I think we can probably drop this extra bit since in this notebook we don't use the term `data pipeline` (we just say pipeline a couple of times)